tests/test_core.py

```diff
@@ -3968,6 +3968,43 @@ def test_dlfcn_feature_in_lib(self):
       '''
     self.do_run(src, 'float: 42.\n')
 
+  @needs_dylink
+  @no_wasm64('TODO: asyncify for wasm64')
+  def test_dlfcn_asyncify(self):
+    self.set_setting('ASYNCIFY')
+    self.set_setting('EXIT_RUNTIME', 1)
+
+    create_file('liblib.c', r'''
+        #include <stdio.h>
+        #include <emscripten/emscripten.h>
+
+        int side_module_run()
+        {
```

Comment:
Can you put open braces on the same line as the declaration?

Also this whole function look like it should be de-indented by 2 spaces.